console.error('Help!')
console.log("Yay, we did it!")
